code/index.html

```diff
@@ -1,30 +1,139 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Stronger Gym</title>
+		<link rel="stylesheet" href="style.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Manrope&family=Open+Sans:wght@400;500&display=swap" rel="stylesheet">
+	</head>
+	<body>
+		<!-- video or image as a header is cool :) -->
+
+<Header>
+
+<div class="textWrapper">
+	
+
+		<h1 class="headerH1"> Athletica Gym</h1>
+		<h3>Stay healthy</h3>
+	
+</div>
+
+<div id="headerWrapper">
+	<video autoplay 
+		autoplay 
+		muted 
+		loop 
+		preload="auto"
+		id="video">
+		<source src="videos/pexels-tima-miroshnichenko-5319759.mp4" type"video/mp4>
+		Your video does not support the video tag
+	  </video>
+</div>
+
+	</Header> 
+
+	<main>		
+		
+		<!-- Signup form -->
+		<section class="wrapperSignIn">
```

Comment:
You might want to take a look at the indentation styling. This is a website I've found helpful: [https://granneman.com/webdev/coding/formatting-and-indenting-your-html](url)

Reply:
I VSC så är det rätt indenterat, oklart varför det ändrar sig här?